src/styles/_variables.scss

```diff
@@ -8,6 +8,8 @@ $WEBEX_MEETING_MIN_WIDTH: 17rem; // Standard small phone width is 20rem, leaving
 // See https://momentum.design/tokens/color
 $md-white: #FFFFFF !default;
 
+$white-alpha-95: rgba(255, 255, 255, 0.95);
```

Comment:
currently, the colors are all prefixed with `md-`. Let's not introduce a new color variable style. You can just use the color code for now in the theme files.